sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergIO.java

```diff
@@ -310,6 +310,13 @@
  *   </tr>
  * </table>
  *
+ * <h3>Runner specific features</h3>
+ *
+ * Google Cloud Dataflow supports additional management features for {@code Managed} including
+ * automatically upgrading transforms to the latest supported version. For more details and
+ * examples, please see <a href="https://cloud.google.com/dataflow/docs/guides/managed-io">Dataflow
+ * managed I/O</a>.
+ *
```

Comment:
I think keeping this text it in Managed is enough? Not sure if it belongs in the IO doc, and to be consistent we'd also need to include it for other Managed IOs

Reply:
Makes sense. Reverted updates to this file.